lutris/game_actions.py

```diff
@@ -391,3 +401,14 @@ def get_displayed_entries(self):
             "add": self.is_installable,
             "view": True
         }
+
+class MultiGameActions(BaseGameActions):
+    def get_multiple_game_actions(self):
```

Comment:
This needs to be named get_game_actions to override the base class method, which it absolutely should be doing.

Reply:
Oh... Missed that. Will override the parent method instead